rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/InfoView.java

```diff
@@ -451,4 +471,11 @@ public void hostSelected(Host h) {
         this.selNS = null;
         this.selNode = null;
     }
+
+    private String[] splitAndExtractNameAndParamsFromDescription(String input) {
+        String[] elements = input.split(",");
```

Comment:
not ideal, this suppose you have a comma after the infrastructure or policy name. 
you should extract the first word of the input instead.
String[] elements = input.split("[,\s]+");